src/msg.jl

```diff
@@ -143,14 +143,18 @@ end
 # adds notation message.field
 
 function Base.getproperty(s::Message, p::Symbol)
```

Comment:
This is now the same as the default implementation, no?

Reply:
Yes, it is. Its been stripped out of all special handling that we can drop the function altogether. Good observation.